forte/orbital-helpers/aosubspace.cc

```diff
@@ -148,173 +141,135 @@ void AOSubspace::startup() {
 
     l_labels_ = {"S", "P", "D", "F", "G", "H", "I", "K", "L", "M"};
 
-    lm_labels_sperical_ = {{"S"},
-                           {"PZ", "PX", "PY"},
-                           {"DZ2", "DXZ", "DYZ", "DX2-Y2", "DXY"},
-                           {"FZ3", "FXZ2", "FYZ2", "FZX2-ZY2", "FXYZ", "FX3-3XY2", "F3X2Y-Y3"},
-                           {"G1", "G2", "G3", "G4", "G5", "G6", "G7", "G8", "G9"},
-                           {"H1", "H2", "H3", "H4", "H5", "H6", "H7", "H8", "H9", "H10", "H11"}};
+    lm_labels_spherical_ = {{"S"},
+                            {"PZ", "PX", "PY"},
+                            {"DZ2", "DXZ", "DYZ", "DX2-Y2", "DXY"},
+                            {"FZ3", "FXZ2", "FYZ2", "FZX2-ZY2", "FXYZ", "FX3-3XY2", "F3X2Y-Y3"},
+                            {"G1", "G2", "G3", "G4", "G5", "G6", "G7", "G8", "G9"},
+                            {"H1", "H2", "H3", "H4", "H5", "H6", "H7", "H8", "H9", "H10", "H11"}};
```

Comment:
@fevangelista Do you prefer putting these labels here or in the header? I don't think this should be changed anyway.